mesheryctl/internal/cli/root/system/system.go

```diff
@@ -45,7 +45,7 @@ var SystemCmd = &cobra.Command{
 		if ok := utils.IsValidSubcommand(availableSubcommands, args[0]); !ok {
 			return errors.New(utils.SystemError(fmt.Sprintf("invalid command: \"%s\"", args[0])))
 		}
-		mctlCfg, err := config.GetMesheryCtl(viper.GetViper())
+		mctlCfg, err := config2.GetMesheryCtl(viper.GetViper())
```

Comment:
Same here on why make it `config2`

Reply:
There is one inside system.go. Do have a look you'll see